manifests/cluster_role_binding.yaml

```diff
@@ -13,4 +13,4 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: observatorium-operator
-  namespace: default
+  namespace: observatorium
```

Comment:
For the e2e tests, we deployed the observatorium-operator in default namespace. the operands will be deployed in `observatorium` namespace. so the clusterRole/ClusterRoleBinding/ServiceAccount should be in `default` namespace.

Reply:
sgtm
revered/pushed/rebased.

Reply:
Thank you